src/ex3/pokemon-client-commander.js

```diff
@@ -0,0 +1,38 @@
+import fetch from "node-fetch";
+import { popularPokemons } from "./popular-pokemons.js";
+
+export class PokemonClient {
+  constructor() {
+    this.API_BASE = 'https://pokeapi.co/api/v2/pokemon';
+    // popular Pokemons list from https://gist.github.com/octalmage/6936761
+  }
+
+  async fetchPokemon(pokemonText) {
+    try {
+      const pokemons = pokemonText.split(',').map( el => el.trim() );
+      let promises = [];
+      pokemons.forEach(pokemon => {
```

Comment:
using .map instead of .forEach will improve readability

Reply:
done